### PR TITLE
WEEK-1 Setup CI

### PR DESCRIPTION
**Pull request by MaksTarnavskyi**

### CODE REVIEW

File: .github/workflows/initial_workflow.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      -
+        name: Build and push simple web-app
+        uses: docker/build-push-action@v2
+        with:
+          context: ./
+          file: week1/Dockerfile
+          builder: ${{ steps.buildx.outputs.name }}
+          push: true
+          tags: ${{ secrets.DOCKER_HUB_USERNAME }}/simple-app:latest
+          cache-from: type=registry,ref=${{ secrets.DOCKER_HUB_USERNAME }}/simple-app:buildcache
```
Comment (truskovskiyk) [inline .github/workflows/initial_workflow.yml:55]: let's move simple-app into parameter too

**Review by truskovskiyk — APPROVED** [review]
@MaksTarnavskyi looks good, please 

- make the build greed
- move simple-app into param

**Comment by MaksTarnavskyi** [comment]
> @MaksTarnavskyi looks good, please
> 
> * make the build greed
> * move simple-app into param

1. moved simple-app into param
2. Build will be green after merging PR1

**Review by truskovskiyk — APPROVED** [review]
@MaksTarnavskyi good job!

**Comment by MaksTarnavskyi** [comment]
Meged and rebased PR1
CI green

image

**Pull request merged**